serf/keymanager.go

```diff
@@ -68,6 +68,10 @@ func (k *KeyManager) streamKeyResp(resp *KeyResponse, ch <-chan NodeResponse) {
 			resp.NumErr++
```

Comment:
Looks like this code will implicitly count the message as implying an error occurred. I'm not sure if that's good or bad - it's an error in the sense that the list produced is incomplete so you can't rely on the output. What's the end-user UX look like? I'm OK if it says its an error and explains but if it for example hides the actual results or say something too dramatic maybe we should reconsider?

Reply:
It is only counted as error if result is false. Since I still return true, even when truncating, it should be fine.
(This was my initial thinking as well and I did not want to make that an error).

Reply:
Oh cool, in that case does our message actually get seen by the user in the CLI output or just logs? Would be nice if it was in CLI output!

Reply:
Well actually, I didn't think of showing it in the UI! But now it even shows up there with https://github.com/hashicorp/consul/pull/5236.